src/components/view/ViewDescription.jsx

```diff
@@ -0,0 +1,102 @@
+"use client";
+import React, { useState } from "react";
+import { FaMinus, FaPlus, FaRegPaperPlane } from "react-icons/fa";
+// import { signIn } from "next-auth/react";
+// import { useSession } from "next-auth/react";
+import Link from "next/link";
+
+const ViewDescription = () => {
+  // const { data: session } = useSession();
+  const { data: session } = useState(true);
```

Comment:
this is illegal code :sob: cause data:session is for Next Auth, u can remove the JS logic for now cause that will change significantly